services/user/justfile

```diff
@@ -1,4 +1,6 @@
 set dotenv-load := false
+venv:
+    if not exist venv ( python -m venv venv && .\venv\Scripts\activate && pip install -r requirements.dev.txt )
```

Comment:
This is highly based on cmd usage... remove! 😠

Reply:
Generalization of just file can be done in a future PR